cmd/releases.go

```diff
@@ -51,3 +56,55 @@ func runReleases(ctx *cli.Context) error {
 
 	return nil
 }
+
+var CmdReleaseCreate = cli.Command{
+	Name:        "create",
+	Usage:       "Create a release on repository",
+	Description: `Create a release on repository`,
+	Action:      runReleaseCreate,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "tag",
+			Usage: "release tag name",
+		},
+		cli.StringFlag{
+			Name:  "target",
+			Usage: "release target refs, branch name or commit id",
+		},
+		cli.StringFlag{
+			Name:  "title, t",
+			Usage: "release title to create",
+		},
+		cli.StringFlag{
+			Name:  "note, n",
+			Usage: "release note to create",
+		},
+		cli.BoolFlag{
+			Name:  "draft, d",
+			Usage: "the release is a draft",
+		},
+		cli.BoolFlag{
+			Name:  "prerelease, p",
+			Usage: "the release is a prerelease",
+		},
+	},
+}
+
+func runReleaseCreate(ctx *cli.Context) error {
+	login, owner, repo := initCommand(ctx)
+
+	_, err := login.Client().CreateRelease(owner, repo, gitea.CreateReleaseOption{
+		TagName:      ctx.String("tag"),
+		Target:       ctx.String("target"),
+		Title:        ctx.String("title"),
+		Note:         ctx.String("note"),
+		IsDraft:      ctx.Bool("draft"),
+		IsPrerelease: ctx.Bool("prerelease"),
+	})
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Shouldn't the user be able to see the error message in cli?

Reply:
Currently log will output to console.